src/python/pants/option/global_options.py

```diff
@@ -251,10 +288,26 @@ def register_bootstrap_options(cls, register):
     register('--build-file-imports', choices=['allow', 'warn', 'error'], default='warn',
       help='Whether to allow import statements in BUILD files')
 
-    register('--remote-store-server',
-             help='host:port of grpc server to use as remote execution file store')
-    register('--remote-execution-server',
-             help='host:port of grpc server to use as remote execution scheduler')
+    remote_store_server = '--remote-store-server'
+    remote_execution_server = '--remote-execution-server'
+    register(remote_store_server, advanced=True,
+             help='host:port of grpc server to use as remote execution file store.')
+    register('--remote-store-thread-count', type=int, advanced=True,
+             default=DEFAULT_EXECUTION_OPTIONS.remote_store_thread_count,
+             help='Thread count to use for the pool that interacts with the remote file store.')
+    register(remote_execution_server, advanced=True,
+             help='host:port of grpc server to use as remote execution scheduler.')
+    register('--remote-store-chunk-size', type=int, advanced=True,
```

Comment:
I'd be tempted to add a `-bytes` to the end of this flag name

Reply:
+1, or just `--remote-store-chunk-bytes`, the "size" being implied.

Reply:
In cases where there is no obvious confusion about what the units could be ("oh, i thought this should be meters!") this seems like overkill. But we've got two votes, so I'll go for it.